TransferMethodRepository/Sources/TransferMethodConfigurationRepository.swift

```diff
@@ -115,18 +115,19 @@ public final class RemoteTransferMethodConfigurationRepository: TransferMethodCo
             guard let strongSelf = self else {
                 return
             }
-            strongSelf.transferMethodConfigurationFieldsCache[fieldQuery] = strongSelf
+            strongSelf.transferMethodConfigurationFieldsCache[fieldQuery] = RemoteTransferMethodConfigurationRepository
                 .performCompletion(error,
                                    result,
                                    completion,
                                    strongSelf.transferMethodConfigurationFieldsCache[fieldQuery])
         }
     }
 
-    private func performCompletion<T>(_ error: HyperwalletErrorType?,
-                                      _ result: T?,
-                                      _ completionHandler: @escaping (Result<T?, HyperwalletErrorType>) -> Void,
-                                      _ repositoryOriginalValue: T?) -> T? {
+    @discardableResult
+    static func performCompletion<T>(_ error: HyperwalletErrorType?,
```

Comment:
I think we may re-use this method across other repositories, so it may be exposed

Reply:
Good point,  Let's discuss Jaspreet during our stand up the idea to have a CommonRepository scheme.  
For me, we could expose this method in a helper class else I would like to have a RepositoryFactory instead to have the <Scheme or module name>RepositoryFactory.swift and inside the schemas (modules) we could create an extension and expose the new repositories.